.github/workflows/pull.yml

```diff
@@ -34,3 +34,8 @@ jobs:
     uses: ./.github/workflows/library_net_tests.yml
     with:
       dafny: ${{needs.getVersion.outputs.version}}
+  pr-interop-test:
+    needs: getVersion
+    uses: ./.github/workflows/library_interop_tests.yml
+    with:
+      dafny: ${{needs.getVersion.outputs.version}}
```

Comment:
As long as we are here, why not do this daily as well?

Reply:
because I was silly and forgot to add it :/